requirements.txt

```diff
@@ -5,3 +5,4 @@ mail-parser
 bs4
 lxml
 eml-parser
+multidecoder
```

Comment:
Since multidecoder is not fully released, I would advise to specify the version. I know there are big plan to refactor a lot in it.